### ISSUE TITLE
val_loss doesn't drop

### ISSUE THREAD

**Issue by easy-and-simple** [zhen8838/K210_Yolo_framework#24]
Hello!
When training loss slowly decrease with each epoch, but val_loss doesn't decrease and slightly vary around some value without any decrease and even can increase a bit.
What is the problem? Does training lower loss as shows loss value or it is not improving as shows val_loss?
